integration/test/ParseObjectTest.js

```diff
@@ -2187,4 +2187,37 @@ describe('Parse Object', () => {
       Parse.allowCustomObjectId = false;
     });
   });
+
+  fit('returns correct field values', async () => {
```

Comment:
Can you remove `fit`, clean up the test and add any additional values you can think of?

Also can you add a unit test to ensure the internals still work this change to JSONArray.
https://github.com/parse-community/Parse-SDK-JS/blob/alpha/src/__tests__/ObjectStateMutations-test.js#L316

Reply:
Please keep `fit` until you have added all the values from and they test passes.